modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/topology/GridDhtPartitionsStateValidator.java

```diff
@@ -329,4 +372,64 @@ private String fold(AffinityTopologyVersion topVer, Map<Integer, Map<UUID, Long>
 
         return sb.toString();
     }
+
+    /**
+     * Folds given map of invalid partition states to string representation
+     * in the following format:
+     * Part [id]: [consistentId=value meta=[updCnt=value, size=value]]
+     * @param topVer Topology version.
+     * @param invalidPartitionsCounters Invalid partitions counters map.
+     * @param invalidPartitionsSize Invalid partitions size map.
+     * @return value is String in the following format: Part [id]:
```

Comment:
I think that the string representing a format should be placed on one line for better understanding.
```
     * @return value is String in the following format:
     *     Part [id]: [consistentId=value meta=[updCnt=value, size=value]]

```